### ISSUE TITLE
Text needs a quote facility, with auto-collapsing of quoted text

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#126]
As requested by a user at:
http://camcycle.cyclescape.org/threads/227#message_4714
and by others previously:

"I am having great difficulty in following this [discussion] in email because Cyclescape removes all the context, so without a lot of effort, I can't tell what URL this is referring to. I'm sure someone has already said, but we need a way of quoting the context without having to think too hard about it."

Obviously replying to an e-mail enables it to be quoted, but the web interface doesn't.

However, when implemented this will need to be done in conjunction with a Google Groups -style parser which auto-hides the quoted text on the website. Otherwise the web view will look a total mess, as it involves scrolling.

Basically I see this as a hard compromise to get right. Web and e-mail are different media; on the web the context is directly above. In e-mail, the context is only present if you sort by Subject line first, which most users won't think to do as they are used to looking down an e-mail to see earlier material.

**Comment by nikolai-b** [comment]
With the richtext editor we have the (block)quote option.

**Comment by mvl22** [comment]
I think this issue is really asking for a 'Reply with quote' link that takes the text of a posting and adds it to the text reply box, quoted.
